UseInstallerEndpoints() does not exist in v14, yet is referenced in the documentation

## What type of issue is it? (Choose one - delete the others)
Wrong documentation

## What article/section is this about?
https://docs.umbraco.com/umbraco-cms/reference/security/ssl-https

## Describe the issue
The endpoint configuration `UseInstallerEndpoints()` that was used in earlier versions of Umbraco is mentioned in the documentation...

<img width="720" alt="Screenshot 2024-06-04 at 11 28 15" src="https://github.com/umbraco/UmbracoDocs/assets/25110864/d06ae2c7-5928-4d91-9f00-f965f335d1ba">

... yet when trying to run Umbraco with a similar implementation and executing `dotnet run`, it throws an error:

> error CS1061: 'IUmbracoEndpointBuilderContext' does not contain a definition for 'UseInstallerEndpoints' and no accessible extension method 'UseInstallerEndpoints' accepting a first argument of type 'IUmbracoEndpointBuilderContext' could be found (are you missing a using directive or an assembly reference?)

What should you use instead? Or maybe simply delete `UseInstallerEndpoints()` from the startup?

--- a/14/umbraco-cms/extending/key-vault.md
+++ b/14/umbraco-cms/extending/key-vault.md
@@ -97,7 +97,6 @@ app.UseUmbraco()
     })
     .WithEndpoints(u =>
     {
-        u.UseInstallerEndpoints();
         u.UseBackOfficeEndpoints();
         u.UseWebsiteEndpoints();
     });

--- a/14/umbraco-cms/reference/routing/custom-routes.md
+++ b/14/umbraco-cms/reference/routing/custom-routes.md
@@ -21,7 +21,6 @@ app.UseUmbraco()
     {
         // This is where to put the custom routing
 
-        u.UseInstallerEndpoints();
         u.UseBackOfficeEndpoints();
         u.UseWebsiteEndpoints();
     });
@@ -232,7 +231,6 @@ app.UseUmbraco()
                 "/shop/{action}/{id?}",
                 new {Controller = "Shop", Action = "Index"});
 
-        u.UseInstallerEndpoints();
         u.UseBackOfficeEndpoints();
         u.UseWebsiteEndpoints();
     });
@@ -364,7 +362,6 @@ app.UseUmbraco()
     {
         u.EndpointRouteBuilder.MapControllers();
 
-        u.UseInstallerEndpoints();
         u.UseBackOfficeEndpoints();
         u.UseWebsiteEndpoints();
     });

--- a/14/umbraco-cms/reference/security/ssl-https.md
+++ b/14/umbraco-cms/reference/security/ssl-https.md
@@ -56,7 +56,6 @@ app.UseHttpsRedirection();
         })
         .WithEndpoints(u =>
         {
-            u.UseInstallerEndpoints();
             u.UseBackOfficeEndpoints();
             u.UseWebsiteEndpoints();
         });

